fe/lib/components/LocationSuggest/LocationSuggestInput.tsx

```diff
@@ -153,6 +174,41 @@ const LocationSuggestInput = ({
           value={locationSuggest}
         />
       </Column>
+      <Column width="content">
+        <Button
+          icon={<IconLanguage />}
+          onClick={() => setShowMapDialog(true)}
+          tone={tone === 'critical' ? tone : undefined}
+          variant="soft"
+        >
+          Select on map
+        </Button>
+        {showMapDialog ? (
```

Comment:
Just wondering, do we still need this ternary?

Reply:
Nope we don't, I'll remove it 👍